### DISCUSSION TITLE
What's the timeline for hail-backend version to be ready?

### DISCUSSION THREAD

**Discussion by SimonLi5601** [broadinstitute/seqr#3062] [Q&A] [answered]
It seems the hail-backend branch has a lot of commits, when it would be expected to be ready, does it require to rewrite the current seqr loading pipeline? Do not see much development on there yet. Thanks!

**Comment by hanars** [comment, marked as answer]
We are hoping to have the new hail backend live in Q1 of 2023 here at the Broad, and then we plan to add support for local installations within a couple months of that.

As you noted, we have made a lot of progress on the actual search code, but have not yet started work on the the necessary updates to the loading pipeline. We will need to update that and add performant architecture to the seqr deployment before we will be able to go live
